portfolio/src/main/webapp/index.html

```diff
@@ -45,15 +45,19 @@ <h2>Blog Posts</h2>
       <div>------------------------------------------------------<div>
 
       <h3>Comment on this Page</h3>
-      <form action="/data" method="POST">
-        <label for="name-input">Name:</label>
-        <textarea id="name-input" name="name-input" rows="1" cols="25">Anonymous</textarea>
-        <br>
-        <p>Comment:</p>
-        <textarea name="comment-input" rows="4" cols="50"></textarea>
-        <br>
-        <input type="submit" />
-      </form>
+      <div id="comment-form">
```

Comment:
Please rebase. I think these were changed in the previous PR, which this PR would overwrite.